server/bin/pbench-backup-tarballs.py

```diff
@@ -105,6 +105,10 @@ def sanity_check(lb_obj, s3_obj, config, logger):
     # make sure the S3 bucket exists
     try:
         s3_obj.head_bucket(s3_obj.bucket_name)
+    except AttributeError:
+        logger.error(
+            "Bucket: not defined in config file")
```

Comment:
Can you make this message a `warning` and add to it saying that S3 backups are not enabled as a result?

Reply:
Sure.